lib/ddtrace/transport/http.rb

```diff
@@ -12,13 +15,12 @@ module Datadog
   module Transport
     # Namespace for HTTP transport components
     module HTTP
-      DEFAULT_AGENT_HOST = '127.0.0.1'.freeze
-      DEFAULT_TRACE_AGENT_PORT = 8126
       DEFAULT_HEADERS = {
-        'Datadog-Meta-Lang'.freeze => Datadog::Ext::Runtime::LANG,
-        'Datadog-Meta-Lang-Version'.freeze => Datadog::Ext::Runtime::LANG_VERSION,
-        'Datadog-Meta-Lang-Interpreter'.freeze => Datadog::Ext::Runtime::LANG_INTERPRETER,
-        'Datadog-Meta-Tracer-Version'.freeze => Datadog::Ext::Runtime::TRACER_VERSION
+        Datadog::Ext::Transport::HTTP::HEADER_CONTAINER_ID => Datadog::Runtime::Container.container_id,
```

Comment:
What happens if this is `nil`? is it excluded from the headers sent to the agent?

Reply:
In this case, no. It would probably send an empty header. Good consideration; might not want to do that, and add a test to verify it doesn't.